pkgs/by-name/ha/halo/package.nix

```diff
@@ -0,0 +1,61 @@
+{ lib
+, stdenv
+, fetchurl
+, makeWrapper
+, temurin-jre-bin
```

Comment:
Why do you use this jre install of default jdk?

Reply:
Because the official is using eclipse's jre. I want to follow the official choice.

Reply:
There is no official openjdk binary so it's reasonable to use temurin jdk upstream. I thought it's better to use the default jdk in distribution unless it doesn't work.

Reply:
I actually want to use the smallest version, I have tried jre_mini but it doesn't come with spring.
I will try again with openjdk.